lib/oli_web/live/system_message_live/edit_message.ex

```diff
@@ -0,0 +1,74 @@
+defmodule OliWeb.SystemMessageLive.EditMessage do
+  use Surface.Component
+
+  alias Oli.Notifications
+  alias OliWeb.Common.FormatDateTime
+  alias Surface.Components.Form
+
+  alias Surface.Components.Form.{
+    Checkbox,
+    DateTimeLocalInput,
+    ErrorTag,
+    Field,
+    HiddenInput,
+    Label,
+    TextArea
+  }
+
+  prop system_message, :struct, required: true
+  prop timezone, :string, required: true
+
+  def render(assigns) do
+    changeset = Notifications.change_system_message(assigns.system_message)
+
+    ~F"""
+      <Form for={changeset} submit="save" class="d-flex align-items-center">
```

Comment:
What do you think about passing the `save` as a required event prop to the component? Because it doesn't implements the `save` by itself